include/irr/asset/filters/CSummedAreaTableImageFilter.h

```diff
@@ -356,20 +384,15 @@ class CSummedAreaTableImageFilter : public CMatchedSizeInOutImageFilterCommon, p
 						normalizeScratch(normalized, asset::isSignedFormat(inFormat));
 
 					{
+						uint8_t* outData = reinterpret_cast<uint8_t*>(state->outImage->getBuffer()->getPointer());
 						const auto blockDims = asset::getBlockDimensions(outFormat);
 
 						auto encode = [&](uint32_t readBlockArrayOffset, core::vectorSIMDu32 readBlockPos) -> void
 						{
-							auto newReadBlockPos = (readBlockPos - decltype(readBlockPos)(state.outOffset.x, state.outOffset.y, state.outOffset.z)); // not sure
+							auto newReadBlockPos = (readBlockPos - decltype(readBlockPos)(state->outOffset.x, state->outOffset.y, state->outOffset.z));
 
 							auto localOutPos = newReadBlockPos * blockDims;
```

Comment:
same as https://github.com/buildaworldnet/IrrlichtBAW/pull/497/files/b296885e7f686a8b2d274882630702e371204feb..cccecdeb1bc53cd049b61c899e128cab9c6cf548?file-filters%5B%5D=.cpp&file-filters%5B%5D=.dds#r472114037

Reply:
corrected in following commit